privaterelay/locales/en-US/app.ftl

```diff
@@ -231,11 +231,9 @@ onboarding-tip-3 = Via the context menu, just right-click (Windows) or Control-c
 modal-rename-alias-saved = Label saved!
 modal-delete-headline = Permanently delete this alias?
 
```

Comment:
Can you make sure there are no empty lined between `modal-delete-headline` and `modal-delete-warning-recovery-html`?